font.js

```diff
@@ -1,9 +1,28 @@
 function setFontFamily(element, data) {
   const fontFamily = data.fontFamily;
+  // Add style to body
+  const styleGlobal = document.createElement("style");
+  styleGlobal.innerHTML = `
+         * {
```

Comment:
Issue. This function creates a new style tag every time a font size is set. Maybe add a check if there is one that already exists, and delete it if it does